sdk/communication/azure-communication-identity/README.md

```diff
@@ -164,11 +164,14 @@ Use the `deleteUser` function to delete a user.
 communicationIdentityClient.deleteUser(user);
 ```
 
-### Exchanging AAD access token of a Teams User for a Communication Identity access token
-Use the `getTokenForTeamsUser` function to exchanges an AAD access token of a Teams User for a new Communication Identity access token.
+### Exchanging Azure AD access token of a Teams User for a Communication Identity access token
+Use the `getTokenForTeamsUser` function to exchanges an Azure AD access token of a Teams User for a new Communication Identity access token.
```

Comment:
```suggestion
Use the `getTokenForTeamsUser` function to exchange an Azure AD access token of a Teams User for a new Communication Identity access token.
```

Reply:
well spotted. @AikoBB can you pls check that we don't have a similar mistake elsewhere?